arangod/Cluster/HeartbeatThread.cpp

```diff
@@ -362,7 +362,7 @@ void HeartbeatThread::run() {
       runSingleServer();
     }
   } else if (ServerState::instance()->isAgent(role)) {
-    runAgent();
```

Comment:
Why is itcorrect to just remove this?

Reply:
because on the agency we don't need a heartbeat thread and the implementation literally does _nothing_.